src/components/generic/editable-field-components/MetaformNumberFieldComponent.tsx

```diff
@@ -0,0 +1,151 @@
+import { FormControl, Input, InputLabel, WithStyles, } from '@material-ui/core';
+import React from 'react';
+import { Metaform, MetaformField } from '../../../generated/client';
+import strings from '../../../localization/strings';
+import styles from "../../../styles/form-edit-screen";
+
+/**
+ * Component props
+ */
+interface Props extends WithStyles<typeof styles> {
+  field: MetaformField;
+  fieldId: string;
+  fieldLabelId: string;
+  metaform: Metaform;
+}
+
+/**
+ * Component state
+ */
+interface State {
+  
+}
+
+/**
+ * Component for Metaform number field
+ */
+export class MetaformNumberFieldComponent extends React.Component<Props, State> {
+
+  /**
+   * Constructor
+   * 
+   * @param props component props
+   */
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+    };
+  }
+
+  /**
+   * Component render method
+   */
+  public render() {
+    if (!this.props.field.name) {
+      return null;
+    }
+
+    /**
+     * Props for number input
+     */
```

Comment:
Jsdoc shouldn't be used here
```suggestion
    // Props for number input
```